File: Java/Important-Programs/05-greatest-of-two-numbers.java
import java.util.Scanner;

class Main {
    public static void main (String[] args) {
        Scanner getInput = new Scanner(System.in);

        System.out.println("Enter first number:");
        int num1 = getInput.nextInt();

        System.out.println("Enter second number:");
        int num2 = getInput.nextInt();

        if (num1 == num2) {
            System.out.println("\nBoth are equal.");
        } else if (num1 > num2) {
            System.out.println("\nGreatest number is: " + num1);
        } 
        else {
            System.out.println("\nGreatest number is: " + num2);
        }

        getInput.close();
    }
}

// Using built-in method
// System.out.println(Math.max(num1, num2));

// Using ternary operator
// System.out.println((num1 > num2) ? num1 : num2);

File: Python/1-Important-Programs/05-greatest-of-two-numbers.py
num1 = int(input("Enter first number: "))
num2 = int(input("Enter second number: "))

if num1 == num2:
    print("Both are equal.")
elif num1 > num2:
    print("\nGreatest number is:", num1)
else:
    print("\nGreatest number is:", num2)

 
# Using pre-built Function
print(max(num1, num2)) 